configure.ac

```diff
@@ -30,7 +30,7 @@ AC_FUNC_STRTOD
 AC_FUNC_FORK
 AC_FUNC_MALLOC
 AC_FUNC_REALLOC
-AC_CHECK_FUNCS([atexit clock_gettime dup2 getopt_long inet_ntoa memmove memset mkdir regcomp rmdir socket strcasecmp strchr strdup strerror strstr strtol strtoul])
+AC_CHECK_FUNCS([atexit clock_gettime dup2 getopt_long memmove memset mkdir regcomp rmdir socket strcasecmp strchr strdup strerror strstr strtol strtoul])
```

Comment:
For consistency with other funcs we should have inet_ntop here, but slirp4netns is Linux-specific anyway, so probably we don’t need to care?